### DISCUSSION TITLE
What's the utility of storing assertions in OpenFGA?

### DISCUSSION THREAD

**Discussion by vhpoet** [openfga/community#66] [Q&A] [answered]
It seems like the workflow of working with assertions is [Writing](https://openfga.dev/api/service#/Assertions/WriteAssertions) them to OpenFGA, and then [Reading](https://openfga.dev/api/service#/Assertions/ReadAssertions) and running a [Check](https://openfga.dev/api/service#/Relationship%20Queries/Check) on each individual one. In a real world scenario I would probably have the assertions and the model stored in a git repo before pushing anything to OpenFGA in which case I can just run [Check](https://openfga.dev/api/service#/Relationship%20Queries/Check) on these assertions without needing to push them to OpenFGA first. Am I missing something obvious here?

**Comment by aaguiarz** [comment, marked as answer]
Hi @vhpoet 

Assertions exist to support Playground-like scenarios, were you don't have another place to store the `check()` calls you want do do.

For a real world scenario, you would run `check()` from your unit tests suite instead of using assertions. 

Does it make sense?

**Reply by vhpoet** [reply]
Makes sense, thank you!
